.idea/$CACHE_FILE$

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
BTW, you can add a global gitignore config on your laptop, this way you won't commit commonly created files by various tools and IDEs